### PR TITLE
Updated : : to - for consistency with the new front-end

### PR DESCRIPTION
**Pull request by ruchikabgosain**
Fixes # 536 from `mozilla/addons`

https://github.com/mozilla/addons/issues/536

CC @diox @caitmuenster

### CODE REVIEW

File: src/olympia/reviewers/templatetags/jinja_helpers.py [hunk 1]
```diff
@@ -56,10 +56,10 @@ def version_status(addon, version):
 def reviewer_page_title(context, title=None, addon=None):
     """Wrapper for reviewer page titles.  Eerily similar to dev_page_title."""
     if addon:
-        title = u'%s :: %s' % (title, addon.name)
+        title = u'%s – %s' % (title, addon.name)
     else:
         section = _('Reviewer Tools')
-        title = u'%s :: %s' % (title, section) if title else section
+        title = u'%s – %s' % (title, section) if title else section
```
Comment (eviljeff) [inline src/olympia/reviewers/templatetags/jinja_helpers.py]: Now addons-server is python3 only we can take the opportunity to remove the `u'...'` prefixes from the strings in this function.

**Comment by diox** [comment]
Also, that doesn't fix the issue at all... consistency is good, but the issue was about giving each page an explicit title (see https://github.com/mozilla/addons/issues/536#issuecomment-456939975)

**Comment by ruchikabgosain** [comment]
@eviljeff  could you help me with the test updation? 
also, @diox hope the titles are cool now?

**Comment by eviljeff** [comment]
I'm going to close this pr for now given it's been almost 2 weeks since we requested changes.  Please open a new pr when/if you have time to continue with it.

**Pull request closed without merging**